### PR TITLE
[XDR - IR] revert #35932

### PR DESCRIPTION
**Pull request by jlevypaloalto**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes:
1. [XSUP-39965](https://jira-dc.paloaltonetworks.com/browse/XSUP-39965)
2. [XSUP-41879](https://jira-dc.paloaltonetworks.com/browse/XSUP-41879)
3. [XSUP-41984](https://jira-dc.paloaltonetworks.com/browse/XSUP-41984)
4. [XSUP-42134](https://jira-dc.paloaltonetworks.com/browse/XSUP-42134)

## Description
Revert version `6.1.73` in #35932.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/CortexXDR/ReleaseNotes/6_1_78.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Palo Alto Networks Cortex XDR - Investigation and Response
+
+- Reverting the de-duplication mechanism added in version `6.1.73`.
```
Comment (YuvHayun) [inline Packs/CortexXDR/ReleaseNotes/6_1_78.md:6]: This RN should be "fixed an issue where - {the issue}"
Additionally, add a note inside 6.1.73 RN that the version is invalid (like the example I sent on slack)

File: Packs/CortexXDR/Integrations/CortexXDRIR/test_data/get_incidents_list_dedup.json [hunk 2]
```diff
@@ -70,52 +70,6 @@
     },
     {
         "incident_id": "4",
-        "creation_time": 100000001,
-        "modification_time": 1575813875168,
```
Comment (YuvHayun) [inline Packs/CortexXDR/Integrations/CortexXDRIR/test_data/get_incidents_list_dedup.json:74]: Is it really necessary to revert here?

**Review by YuvHayun — APPROVED** [review]
Looks good.
approved for my side.
Just note the comments about the RN.
Make sure @JudahSchwartz also review it.

File: Packs/CortexXDR/Integrations/CortexXDRIR/test_data/get_incidents_list_dedup.json (hunk 2 above)
Comment (jlevypaloalto) [reply Packs/CortexXDR/Integrations/CortexXDRIR/test_data/get_incidents_list_dedup.json:74]: it will make it easier for me to unrevert

**Review by JudahSchwartz — APPROVED** [review]

**Pull request merged**